### PR TITLE
#1425 KSQL throws NPE if no preceding space before line continuation char

### LINKED ISSUE #1425: KSQL throws NPE if no preceeding space before line continuation character
```
ksql> SELECT MESSAGE\
> FROM RATINGS;io.confluent.ksql.parser.exception.ParseFailedException
Caused by: java.lang.NullPointerException
ksql> SELECT MESSAGE \
> FROM RATINGS;
is this as good as it gets? really ?
your team here rocks!
```

Related: #1424

### PR DESCRIPTION
**Pull request by uurl**
### Description 
This PR fixes #1425 KSQL throws NPE if no preceding space before line continuation char

### Testing done 
mvn clean package

### Reviewer checklist
- [x] Ensure docs are updated if necessary. (eg. if a user visible feature is being added or changed).
- [x] Ensure relevant issues are linked (description should include text like "Fixes #<issue number>")

### CODE REVIEW

**Comment by uurl** [comment]
@big-andy-coates @apurvam @rodesai review please

File: ksql-cli/src/test/java/io/confluent/ksql/CliTest.java [hunk 1]
```diff
@@ -445,6 +445,11 @@ public void testEmptyInput() throws Exception {
     localCli.runNonInteractively("");
   }
 
+  @Test
+  public void testContinuation() throws Exception{
+    localCli.runNonInteractively("SHOW" + "\\\t \t\n"  + "TABLES;");
```
Comment (apurvam) [inline ksql-cli/src/test/java/io/confluent/ksql/CliTest.java]: Shouldn't this test case be simply `SHOW\nTABLES` to reproduce the original bug?

Comment (uurl) [reply ksql-cli/src/test/java/io/confluent/ksql/CliTest.java]: @apurvam ready 👍

**Comment by big-andy-coates** [comment]
Hi @uurl,

While looking into reviewing this issue for you and working out what is needed, I also worked out what the fix was, so I've gone ahead and added it here: #1829.

Hence, I'm going to close this PR in favour of my own. However, still a big 👍 for your contribution!

**Pull request closed without merging**